jenkinsapi/build.py

```diff
@@ -91,6 +91,16 @@ def get_revision_branch(self):
         vcs = self._data['changeSet']['kind'] or 'git'
         return getattr(self, '_get_%s_rev_branch' % vcs, lambda: None)()
 
+    def get_params(self):
+        """
+        Return a dictionary of params names and their values or None
+        if no parameters present
+        """
+        actions = self._data.get('actions')
+        if actions:
+            parameters = actions[0].get('parameters', {})
```

Comment:
This does not work in my case, since the 'parameters' obj is not at index 0 of the actions obj but at index 1. actions[0] contains the 'causes' object